adapter/internal/oasparser/envoyconf/http_filters.go

```diff
@@ -246,7 +249,7 @@ func getLuaFilter() *hcmv3.HttpFilter {
 		logger.LoggerOasparser.Error(err2)
 	}
 	luaFilter := hcmv3.HttpFilter{
-		Name: wellknown.Lua,
+		Name: filterName,
```

Comment:
isn't this always `envoy.filters.http.lua` (you have provided custom values right?)

Reply:
Since we now have two lua filters, there should be two unique names. LuaPerRoute configs refers to these names for overriding.